src/gui/src/AboutDialog.h

```diff
@@ -22,17 +23,17 @@
 
 #include <QDialog>
 
-#include "ui_AboutDialogBase.h"
-
-class QWidget;
-class QString;
+namespace Ui
+{
+    class AboutDialog;
+}
 
-class AboutDialog : public QDialog, public Ui::AboutDialogBase
+class AboutDialog : public QDialog
 {
     Q_OBJECT
-
     public:
         AboutDialog(QWidget* parent, const QString& app_name);
+    private:
+        Ui::AboutDialog *ui = nullptr;
```

Comment:
`*` should positioned near the type, not variable. Please also fix all other occurrences.

Reply:
Is there a code style guide for this project ? , If not lets get one so we don't have to waste time on these style issues. We could also use a git hook to run clang-format or similar to format the code the to our preferred format.